ros/lib/config.py

```diff
@@ -69,3 +72,10 @@ def get_logger(name):
 # Number of days after which data is considered to be outdated.
 DAYS_UNTIL_STALE = os.getenv("DAYS_UNTIL_STALE", 14)
 INSTANCE_PRICE_UNIT = 'USD/hour'
+
+if CW_ENABLED is True:
+    # Available only in k8s namespace, through an app-interface automation
+    AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY_ID", None)
+    AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY", None)
+    AWS_REGION_NAME = os.getenv("AWS_REGION_NAME", None)
+    AWS_LOG_GROUP = os.getenv("AWS_LOG_GROUP", None)
```

Comment:
Aren't these veriables required for Clowder Enabled state too?

Reply:
@apuntamb Yes, in clowder if else block I'm setting up the CW_ENABLED flag and then finally based on the flag I'm init-ing these values

Reply:
@saltgen You mean these variables should already be set (we don't have to set the explicitly) if `CW_ENABLED = True` else default value will be `None`

Reply:
`AWS_ACCESS_KEY_ID=app_common_python.LoadedConfig.logging.cloudwatch.accessKeyId`
Like this one ^^

Reply:
@apuntamb @r14chandra Thanks for the tip(clowder-native config values) :raised_hands: 
Changes pushed :rocket: